nvim-client-0.0.1-20.rockspec

```diff
@@ -12,7 +12,8 @@ dependencies = {
   'lua ~> 5.1',
   'lua-messagepack',
   'luv',
-  'coxpcall'
+  'coxpcall',
+  'luaposix',
```

Comment:
There's no need to add it as a rockspec dependency since it is only used for testing